docs/api-docs/analytics/bodl-overview.md

```diff
@@ -0,0 +1,462 @@
+
+## Standard BODL schema with Stencil object values
+
+The example [initialization script](#initialize-bodl) for the standard `BODL` schema adds properties and methods in two ways. First, it captures a set of values from Stencil objects. Then, it adds methods and constructs several additional properties using the Stencil values.
+
+Consult [Stencil object reference](/theme-objects) for more about object properties. In a headless environment, the GraphQL Storefront API can return similar information.
+
+| BODL Property | Stencil Object |
+|:---|:---|
+| breadcrumbs | [{{breadcrumbs}}](/theme-objects/breadcrumbs) |
+| brand | [{{brand}}](/theme-objects/brand) |
+| cartId | [{{cart_id}}](/theme-objects/cart_id) |
+| cartItemAdded | [{{cart.added_item}}](/theme-objects/carts) |
+| categoryProducts | [{{category.products}}](/theme-objects/category) |
+| categoryName | [{{category.name}}](/theme-objects/category) |
+| order | [{{order}}](/theme-objects/order) |
+| productId | [{{product.id}}](/theme-objects/product) |
+| productTitle | [{{product.title}}](/theme-objects/product) |
+| productCurrency | [{{product.price.without_tax.currency}}](/theme-objects/product) |
+| productPrice | [{{product.price.without_tax.value}}](/theme-objects/product) |
+| products | [{{products}}](/theme-objects/products) |
+| search | [{{product_results}}](/theme-objects/product_results) |
+| wishlist | [{{wishlist}}](/theme-objects/wishlist) |
+
+
+A standard `BODL` instance contains the following calculated properties:
+
+| BODL Property | First-Level Child Properties | 
+|:---|:---|
+| category: object | name: string, products: array |
+| product: object | id: string, title: string, price: object |
+
+A standard `BODL` instance contains the following methods:
+
+| BODL Method | Parameter(s) | Parameter Properties | Return Value |
+|:---|:---|:---|:---|
+| getCartItemContentId | `item`: object | `item.type`: string, `item.product_id`: string | If the item is a gift certificate, returns the item's `type`. Otherwise, returns the item's `product_id`.  |
+| getQueryParamValue | `name`: string | **X** | Returns the value of the specified query parameter from the window's URL. |
+
+## Initialize BODL
+
+The following script extracts storefront data from the Stencil objects available in the front-end environment to construct a standard `BODL` object.
+
+```handlebars title="Sample Script Code Start: Initialization Script & Page Event" lineNumbers
+<script>
+  
+  if (typeof BODL === 'undefined') {
+    // https://developer.bigcommerce.com/theme-objects/breadcrumbs
+    {{inject "breadcrumbs" breadcrumbs}}
+    // https://developer.bigcommerce.com/theme-objects/brand
+    {{inject "brand" brand}}
+    // https://developer.bigcommerce.com/theme-objects/category
+    {{inject "categoryProducts" category.products}}
+    {{inject "categoryName" category.name}}
+    {{inject "customerId" customer.id}}
```

Comment:
Can you add a comment here for the customer object, like we do for the other injected variables. Just to be consistent.

https://developer.bigcommerce.com/theme-objects/customer